trace/spec/semantic_conventions.md

```diff
@@ -6,36 +6,38 @@ The **Semantic Conventions** define the keys and values which describe commonly
-| `exception.message`             | String          | `"Null value encountered"`                       |                   | Detailed message describing the exception                        |
-| `exception.escaped`             | Boolean         | `true`                                           |                   | Indicator if the exception has escaped the span's scope          |
-| `exception.stacktrace`          | String          | `"at app.main(app.java:16)"`                     |                   | The stack trace of the exception                                 |
-| `output.value`                  | String          | `"Hello, World!"`                                |                   | The output value of an operation                                 |
-| `output.mime_type`              | String          | `"text/plain"`                                   |                   | MIME type representing the format of `output.value`              |
-| `input.value`                   | String          | `{"query": "What is the weather today?"}`        | Required          | The input value to an operation                                  |
-| `input.mime_type`               | String          | `"application/json"`                             |                   | MIME type representing the format of `input.value`               |
-| `embedding.embeddings`          | List of objects | `[{vector: [...], text: "hello"}]`               |                   | List of embedding objects including text and vector data         |
-| `embedding.model_name`          | String          | `"BERT-base"`                                    |                   | Name of the embedding model used                                 |
-| `embedding.text`                | String          | `"hello world"`                                  |                   | The text represented in the embedding                            |
-| `embedding.vector`              | List of floats  | `[0.123, 0.456, ...]`                            |                   | The embedding vector consisting of a list of floats              |
-| `llm.function_call`             | String          | `{function_name: "add", args: [1, 2]}`           |                   | Object recording details of a function call in models or APIs    |
-| `llm.invocation_parameters`     | JSON string     | `{model_name: "gpt-3", temperature: 0.7}`        |                   | Parameters used during the invocation of an LLM or API           |
-| `llm.messages`                  | List of objects | `[{role: "user", content: "hello"}]`             |                   | List of messages exchanged in a chat API                         |
-| `message.role`                  | String          | `"user"` or `"system"`                           |                   | Role of the entity in a message (e.g., user, system)             |
```

Comment:
Worth enumerating the possible values here?

Reply:
Probably not since it's a string and I'm not sure OpenAI has the final say here